valhalla/settings.py

```diff
@@ -108,6 +108,10 @@
     'default': {
         'BACKEND': os.getenv('CACHE_BACKEND', 'django.core.cache.backends.dummy.DummyCache'),
         'LOCATION': os.getenv('CACHE_LOCATION', 'unique-snowflake')
+    },
+    'configdb': {
```

Comment:
Can we rename this to 'locmem' or something? There's no reason why we couldn't use it for something else in the future.